scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Split.scala

```diff
@@ -5,6 +5,8 @@ import scala.meta.tokens.Token
 import org.scalafmt.internal.Policy.NoPolicy
 import org.scalafmt.util.PolicyOps
 import org.scalafmt.util.TokenOps
+import org.scalameta.FileLine
+import java.nio.file.Paths
 
```

Comment:
why do you need Paths?

Reply:
I don't really need it now, thanks! Removed